src/main/java/de/adesso/anki/battle/protocol/commands/FireRocketCommand.java

```diff
@@ -3,6 +3,7 @@
 import de.adesso.anki.battle.world.World;
 import de.adesso.anki.battle.world.bodies.Rocket;
 import de.adesso.anki.battle.world.bodies.Vehicle;
+import de.adesso.anki.sdk.messages.VehicleStateUpdateMessage;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'de.adesso.anki.sdk.messages.VehicleStateUpdateMessage'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=34&issues=AWStlUM26dA5b-W2mZp2&open=AWStlUM26dA5b-W2mZp2)

[ISSUE_KEY]: AWStlUM26dA5b-W2mZp2